cmd/kubeadm/app/master/manifests.go

```diff
@@ -370,25 +356,19 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) [
 	return command
 }
 
-func getControllerManagerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) []string {
-	var command []string
-
-	// self-hosted controller-manager needs to wait on a lock
-	if selfHosted {
-		command = []string{"/usr/bin/flock", "--exclusive", "--timeout=30", "/var/lock/controller-manager.lock"}
```

Comment:
The commit message suggests there is some issue to do with a lock, but it doesn't clarify what the issue is. Ideally there should be an issue ticket for it, or at very least detailed description in the commit message.